src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -43,10 +45,27 @@
 /**
  * Extends The Symfony framework bundle controller to add common functions for PrestaShop needs.
  */
-class FrameworkBundleAdminController extends AbstractController
+class FrameworkBundleAdminController extends AbstractController implements ContainerAwareInterface
```

Comment:
Add deprecated notice (without fixed version for removal) to explain that we should favor PrestaShopAdminController instead. This class will be removed in a major version (at least 10) once all the usage of this class have been removed from the core